dist/trip.js

```diff
@@ -1056,6 +1056,26 @@
         'e s w n screen-ne screen-se screen-sw screen-nw screen-center');
       $tripBlock.addClass(o.position);
 
+      // if we have a nextClickSelector use that as the trigger for the next button
+      if(o.nextClickSelector){
+        var eventName = (o.nextClickEvent || 'click')+'.Trip'
+        $(o.nextClickSelector).one(eventName, function(e) {
```

Comment:
I think we should try to extract logics out from `isTripDataValid` to make sure we can double check passing `nextClickSelector` (Like the element doesn't exist ... blah)